src/components/ProcessList/ProcessList.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { toWords } from 'number-to-words';
+
+export default class ProcessList extends Component {
+  static propTypes = {
+    children: PropTypes.node.isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const { children } = this.props;
+
+    return (
+      <div className="process schemaform-process">
+        <ol role="list">
+          {children.map((child, index) => (
```

Comment:
What happens if there's only one child passed to this component? (Would be weird for a process list, but :man_shrugging:) We might want to use [`React.Children.map`](https://reactjs.org/docs/react-api.html#reactchildrenmap) instead.

Reply:
Thanks for reviewing. Good catch, nothing renders when there's only one child. It's working with `React.Children`.